src/etl/construct_etl.py

```diff
@@ -134,168 +121,196 @@ def _process_sub_type(self, sub_type):
-                "loadKey": loadKey,
+                "symbol": construct_record.get('name'),
+                "primaryId": construct_record.get('primaryId'),
+                "globalId": global_id,
+                "localId": local_id,
+                "dataProviders": data_providers,
+                "dateProduced": date_produced,
+                "loadKey": load_key,
                 "release": release,
-                "modGlobalCrossRefId": modGlobalCrossRefId,
+                "modGlobalCrossRefId": mod_global_cross_ref_id,
                 "uuid": str(uuid.uuid4()),
                 "dataProvider": data_provider,
-                "nameText": nameText
+                "nameText": name_text
             }
             constructs.append(construct_dataset)
 
-            if 'crossReferences' in constructRecord:
+            if 'crossReferences' in construct_record:
 
-                for crossRef in constructRecord['crossReferences']:
-                    crossRefId = crossRef.get('id')
-                    local_crossref_id = crossRefId.split(":")[1]
-                    prefix = crossRef.get('id').split(":")[0]
-                    pages = crossRef.get('pages')
+                for cross_ref in construct_record['crossReferences']:
+                    cross_ref_id = cross_ref.get('id')
+                    local_crossref_id = cross_ref_id.split(":")[1]
+                    prefix = cross_ref.get('id').split(":")[0]
+                    pages = cross_ref.get('pages')
 
                     # some pages collection have 0 elements
                     if pages is not None and len(pages) > 0:
                         for page in pages:
                             if page == 'construct':
-                                modGlobalCrossRefId = ETLHelper.get_page_complete_url(local_crossref_id, self.xrefUrlMap, prefix, page)
-                                xref = ETLHelper.get_xref_dict(local_crossref_id, prefix, page, page, crossRefId, modGlobalCrossRefId, crossRefId+page)
-                                xref['dataId'] = globalId
-                                crossReferenceList.append(xref)
-
-            if 'constructComponents' in constructRecord:
-                for component in constructRecord.get('constructComponents'):
-                    componentRelation = component.get('componentRelation').upper()
-                    componentSymbol = component.get('componentSymbol')
-                    componentID = component.get('componentID')
-
-                    if componentID is not None:
-                        componentDetail = {
-                            "componentRelation": componentRelation.upper(),
-                            "componentSymbol": componentSymbol,
-                            "componentID": componentID,
-                            "constructID": constructRecord.get('primaryId')
+                                mod_global_cross_ref_id = ETLHelper.get_page_complete_url(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \.